src/Microsoft.AspNetCore.Server.Kestrel/Internal/Http/IHttpHeadersHandler.cs

```diff
@@ -0,0 +1,9 @@
+using System;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Internal.Http
+{
+    public interface IHttpHeadersHandler
```

Comment:
As aside shame you can't mark interfaces as constraint only; would save accidentally boxing worries with structs